multinetjs/src/index.ts

```diff
@@ -1,13 +1,18 @@
 import { Client } from './client';
 
+export interface TableRow {
+  _key: string;
+  _id: string;
+}
+
```

Comment:
Would this ever break? Can the TableRow have more than just _key and  _id? Will it have attributes?

Reply:
The beauty of TypeScript "interfaces" is they're composable. That is, saying that a value is a `TableRow` means that it must have `_key` and `_id` keys, but can also have other things in it as well. If you had a specific type that just encoded someone's name and age, you could do:

```typescript
interface Person {
  name: string;
  age: number;
}
```

and then declare a specific table's rows as having type `TableRow & Person`. But in general, a `TableRow` can indeed have other attributes on it.

Reply:
Awesome, thanks for the clarification